### ISSUE TITLE
7900 XTX Pytorch Issues

### ISSUE THREAD

**Issue by tedtroxell** [ROCm/ROCm#1977]
Like a few others who have posted here I have a 7900 XTX, which isn't officially supported by the ROCm stack. I've looked on line, but I haven't found any information on when to expect support for that device. I'm currently using PyTorch and although I can install torch so it recognizes that there is a GPU, it segfaults whenever a tensor is placed onto the gpu similar to the issue submitted [here](https://github.com/RadeonOpenCompute/ROCm/issues/1973).

I find it disappointing that the 7900 XTX was released 3 months ago and 3 versions of ROCm 5.4.x have been released without support for this.

Is there any reasonably stable way to get pytorch to run on a 7900 XTX? If not, how long are we expected to wait to be able to do so?

**Comment by KylinC** [comment]
I went to the same situation, but I can put the tensor on RX7900xtx, but segment fault when I operated it (add, multiply). I tried to build a docker, or build pytorch from source, but all crashed.

**Comment by littlewu2508** [comment]
There are some discussions in https://github.com/RadeonOpenCompute/ROCm/issues/1880, including trying pytorch

**Comment by hongxiayang** [comment]
@tedtroxell Please check below documentation about how to install pytorch/ROCm on 7900XTX.

https://rocm.docs.amd.com/projects/radeon/en/latest/index.html

**Comment by briansp2020** [comment]
@hongxiayang 
What is the difference in installing ROCm using
1. https://repo.radeon.com/amdgpu-install/23.20.00.48/ubuntu/jammy/amdgpu-install_5.7.00.48.50700-1_all.deb 
2. https://repo.radeon.com/amdgpu-install/5.7/ubuntu/jammy/amdgpu-install_5.7.50700-1_all.deb
3. https://rocm.docs.amd.com/en/latest/deploy/linux/quick_start.html
Not sure why there are 2 different amdgpu-install script with different version number.

**Comment by hongxiayang** [comment]
Please use 1) instead of 2) for your case because of the first one has the power management fix.

**Comment by briansp2020** [comment]
@hongxiayang
If amdgpu-install_5.7.00.48.50700-1_all.deb has more fixes, does it mean that [amdgpu-install_5.7.50702-1_all.deb](https://repo.radeon.com/amdgpu-install/5.7.2/ubuntu/jammy/amdgpu-install_5.7.50702-1_all.deb), which was released later, have the fix as well?

Anyway, my question is not which one to use. But why have two different version number schemes 23.20.00.48 and 5.7? It just seems confusing for no good reason.

**Comment by hongxiayang** [comment]
cc @saadrahim

**Comment by briansp2020** [comment]
https://rocm.docs.amd.com/en/latest/deploy/linux/installer/install.html

5.7.1 is the official version number scheme that ROCm has been using as long as I've been following ROCm development and I've been following it ever since 1.0 was released. I saw 23.20.00.02 a few weeks ago on Twitter when support for 7900XTX was announced. Though now I know where it came from now (it is the version number scheme the graphics driver team uses, right?), it was very confusing when I first saw it (https://twitter.com/briansp2020/status/1714068351176020310), and felt that it was unnecessary and showed a lack of planning and communication within AMD about how to bring ROCm to consumer products. I'm pretty sure most people who have been following ROCm development for more than a year are more familiar with the 5.7.1 format and would be confused by the 23.20.00.02 scheme.
Anyway, I think AMD needs 1 version number scheme that rules them all or, at least, should communicate clearly that there is an ROCm distribution version number and a graphics driver version number and how they relate to each other. Or maybe ROCm distribution version should be treated separately as has been the case so far and only mention the 23.20... as the version number of the script that installs ROCm components that matches the driver distribution.
just my 2c

**Comment by briansp2020** [comment]
https://repo.radeon.com/amdgpu-install/

I see that both version number's been in use for a while.  I think AMD should
1. stick to ROCm version number only
or
1. release 1 script with both ROCm and driver version number. 
2. clearly indicate that ROCm version number and GPU driver version numbers are two different things and how they should always go together.

NVidia has a separate driver version number and CUDA version number as well. but they don't use the driver version when talking CUDA and they don't talk CUDA version number when talking about driver release. The twit I saw was talking about a ROCm release but mentioned the driver version, which is why I got confused. So, I think this is more of a communication/documentation issue than anything else.
